packages/cms-cli/commands/secrets/addSecret.js

```diff
@@ -21,12 +21,13 @@ const {
   getPortalId,
 } = require('../../lib/commonOpts');
 const { logDebugInfo } = require('../../lib/debugInfo');
+const { secretValuePrompt } = require('../../lib/secretPrompt');
 
-exports.command = 'add <name> <value>';
+exports.command = 'add <name>';
```

Comment:
I think we also need to add the positional argument from the builder.

Reply:
I went ahead and removed those